dockerfiles/cli/scripts/openshift/deploy_che.sh

```diff
@@ -115,7 +127,12 @@ if [ "${OPENSHIFT_FLAVOR}" == "minishift" ]; then
   CHE_OPENSHIFT_PROJECT=${CHE_OPENSHIFT_PROJECT:-${DEFAULT_CHE_OPENSHIFT_PROJECT}}
   DEFAULT_OPENSHIFT_NAMESPACE_URL="${CHE_OPENSHIFT_PROJECT}.$(minishift ip).nip.io"
   OPENSHIFT_NAMESPACE_URL=${OPENSHIFT_NAMESPACE_URL:-${DEFAULT_OPENSHIFT_NAMESPACE_URL}}
-  DEFAULT_CHE_KEYCLOAK_DISABLED="true"
+  if [ "$CHE_MULTI_USER" == "true" ]
```

Comment:
i'm not sure we need to add this check as part of the switch on the flavor, it can be before the switch